deployment/deployer.yaml

```diff
@@ -148,4 +172,51 @@ parameters:
   description: "Timeout for *expected* ops nodes to be present when cluster is recovering from a full restart."
   name: ES_OPS_RECOVER_AFTER_TIME
   value: "5m"
-
+-
+  description: "Node selector key for Elasticsearch cluster (key: value)."
+  name: ES_NS_KEY
```

Comment:
with using the sed approach, if I have the user pass in a single variable as `type=infra`, the result after the `oc process` is `"nodeSelector": "type=infra"` instead of the expected:
```
nodeSelector:
    type: infra
```

Or am I incorrect in thinking those two are different?